Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -236,7 +248,7 @@ def start_streaming_exp(self, experiment_parameters, mouse1, mouse2, opponent_pa
         self.trial_logger.start_logging(experiment_parameters.get("mouse_id"), opponent_path)
         self.event_logger.start_logging(experiment_parameters.get("mouse_id"), opponent_path)
         self.num_trial = experiment_parameters.get("num_trials")
-
+        force_end_time = self.num_trial * (experiment_parameters.get("decision_time") + experiment_parameters.get("return_time"))
```

Comment:
You brought the force_end_time back. I though you decided you did not need it any more. What Happened? Also, it seams to me that the variable is not being used at all.

Reply:
its not needed .i dont know how it made iits way back.ill remove it